README.md

```diff
@@ -26,17 +26,19 @@ This package is currently in the active development.
 ## Requirements
 
 * [PHP 7.2](http://php.net/releases/7_2_0.php) or greater
-* [Symfony 4](https://symfony.com/4) or [Symfony 3.4](https://symfony.com/roadmap/3.4)
+* [Symfony 4.2](https://symfony.com/roadmap/4.2) or [Symfony 3.4](https://symfony.com/roadmap/3.4)
 
 ## Installation
 
-1. Require the bundle with Composer:
+1. Require the bundle and a PSR 7/17 implementation with Composer:
 
     ```sh
-    composer require trikoder/oauth2-bundle --no-plugins --no-scripts
+    composer require trikoder/oauth2-bundle nyholm/psr7 --no-plugins --no-scripts
     ```
 
-    > **NOTE:** Due to required pre-configuration, this bundle is currently not compatible with [Symfony Flex](https://github.com/symfony/flex).
+    > **NOTE #1:** Due to required pre-configuration, this bundle is currently not compatible with [Symfony Flex](https://github.com/symfony/flex).
+
+    > **NOTE #2:** This bundle requires a PSR 7/17 implementation to operate. We recommend that you use [nyholm/psr7](https://github.com/Nyholm/psr7). Check out this [document](docs/psr-implementation-switching.md) if you with to use a different implementation. 
```

Comment:
@spideyfusion "With" -> "wish" typo 😛 The rest of the PR looks good (as we had discussed) 👍

Reply:
@X-Coder264 Good catch.